packages/shared/src/url.ts

```diff
@@ -35,29 +35,26 @@ export function addClerkPrefix(str: string | undefined) {
  * param or use the frontendApi to determine if the canary tag should be used.
  * The default tag is `latest`.
  */
-export const getClerkJsMajorVersionOrTag = (frontendApi: string, pkgVersion?: string) => {
-  if (!pkgVersion && isStaging(frontendApi)) {
+export const getClerkJsMajorVersionOrTag = (frontendApi: string, version?: string) => {
+  if (!version && isStaging(frontendApi)) {
     return 'canary';
   }
 
-  if (!pkgVersion) {
+  if (!version) {
     return 'latest';
   }
 
-  return pkgVersion.split('.')[0] || 'latest';
+  return version.split('.')[0] || 'latest';
 };
 
 /**
  *
  * Retrieve the clerk-js script url from the frontendApi and the major tag
  * using the {@link getClerkJsMajorVersionOrTag} or a provided clerkJSVersion tag.
  */
-export const getScriptUrl = (
-  frontendApi: string,
-  { pkgVersion, clerkJSVersion }: { pkgVersion?: string; clerkJSVersion?: string },
-) => {
+export const getScriptUrl = (frontendApi: string, { clerkJSVersion }: { clerkJSVersion?: string }) => {
   const noSchemeFrontendApi = frontendApi.replace(/http(s)?:\/\//, '');
-  const major = getClerkJsMajorVersionOrTag(frontendApi, pkgVersion);
+  const major = getClerkJsMajorVersionOrTag(frontendApi, clerkJSVersion);
   return `https://${noSchemeFrontendApi}/npm/@clerk/clerk-js@${clerkJSVersion || major}/dist/clerk.browser.js`;
```

Comment:
@nikosdouvlis Since we are dropping the `pkgVersion` we won't be able to load the major version of a version provided implicitly because the `clerkJSVersion || major` where the `major` version is being calculated using the `clerkJSVersion`.
This is the reason behind dropping the test in `packages/shared/src/__tests__/url.test.ts`.

We will be able to use:
- the `latest` or `canary` tag or
- the explicit version passed

Reply:
@nikosdouvlis  the `clerk-react` does not use this script to load the `clerk-js`. This is used to load ClerkJS in interstitial and it seems to be an existing issue that needs to be solved.
I will merge this an open a ticket to validate and handle the clerkJS version loaded in the interstitial